extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -177,14 +209,30 @@ const renderOutlinkPageAttachmentUI = (
       'i-amphtml-story-outlink-page-attachment-outlink-chip-no-image'
     );
   } else {
-    const ctaImgEl = win.document.createElement('div');
-    ctaImgEl.classList.add('i-amphtml-story-outlink-page-attachment-img');
     if (openImgAttr) {
+      const ctaImgEl = win.document.createElement('div');
+      ctaImgEl.classList.add('i-amphtml-story-outlink-page-attachment-img');
       setImportantStyles(ctaImgEl, {
         'background-image': 'url(' + openImgAttr + ')',
       });
+      chipEl.prepend(ctaImgEl);
+    } else {
+      const ctaLinkImageEl = win.document.createElement('div');
```

Comment:
This element doesn't seem to be used at all, you can use below any other element to create the HTML template (attachmentEl should work)